lib/glossary.typ

```diff
@@ -41,7 +41,12 @@
   if data.len() > 0 [
     #heading(title) #label("glossary")
   ]
-  for (word, info) in data {
-    [/ #word: #info.definition\ → #if "link" in info {link(info.link)}]
-  }
-}
\ No newline at end of file
+  
+
+  (terms(..data.pairs().map(((word,info)) => terms.item(word, [#info.definition\ → #if "link" in info {link(info.link)}]))))
+
+}
+
+
+
+
```

Comment:
nit: remove extra newlines, leave only one newline at the very end

Reply:
I'll fix these. I didn't reference the PR in the commit message. Should I just mess with the ```changelog.csv``` file so things don't look messy?

Reply:
`changelog.csv` is auto generated from the commit history. When a PR gets merged, it'll get squashed into a single commit, and it'll reference the PR in the message. No action needed.